chemicaltoolbox/autodock_vina/prepare_box/prepare_box.xml

```diff
@@ -0,0 +1,37 @@
+<tool id="prepare_box" name="Calculate the box parameters for an AutoDock Vina job from an input mol file (confounding box)" version="0.1.0">
+    <requirements>
+        <requirement type="package">rdkit</requirement>
+        <requirement type="package">numpy</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python $__tool_directory__/calc_vina_box_params.py --ligand '$input1' --config '$output1' --bufx 1 --bufy 2 --bufz 3
```

Comment:
please also put tool_directory/calc... in single quotes. It can be that in some galaxy installation there is a space in the path ... who knows ;)

Reply:
resolved by c0d9ecf83a9a57b4011d4e47769ed950fd5ee798